spec.md

```diff
@@ -3767,6 +3912,278 @@ x.remove(2)                             # None (x == [1, 3])
+
+If `s` and `t` are sets, `s.difference_update(t)` is equivalent to `s -= t`;
+however, note that the `-=` augmented assignment requires both sides to be sets,
+while the `difference_update` method takes arbitrary iterable sequences.
+
+It is permissible to call `difference_update` without any arguments; this leaves
+the set `S` unchanged.
+
+`difference_update` fails if the set `S` is frozen or has active iterators.
+
+```python
+s = set([1, 2, 3, 4])
+s.difference_update([2])             # None; s is set([1, 3, 4])
+s.difference_update([0, 1], [4, 5])  # None; s is set([3])
+```
+
+<a id='set·discard'></a>
+
+### set·discard
+
+`S.discard(x)` removes the value `x` from the set `S` if `x` is present in `S`.
+It returns `None`.
+
+It is permissible to `discard` a value not present in the set; this leaves the
+set `S` unchanged. If you want to fail on an attempt to remove a non-present
+element, use [`remove`](#set·remove) instead. If you need to remove multiple
+elements from a set, see [`difference_update`](#set·difference_update) or the
+[`-=`](#sets) augmented assignment operation.
+
+`discard` fails if the set `S` is frozen or has active iterators.
+
+```python
+s = set(["x", "y"])
+s.discard("y")  # None; s == set(["x"])
+s.discard("y")  # None; s == set(["x"])
+```
+
+<a id='set·intersection'></a>
+
+### set·intersection
+
+`S.intersection(*others)` returns a new set containing elements found in the set
+`S` but not found in any of the iterable sequences `*others`.
+
+If `s` and `t` are sets, `s.intersection(t)` is equivalent to `s & t`; however,
+note that the `&` operation requires both sides to be sets, while the
+`intersection` method takes arbitrary iterable sequences.
+
+It is permissible to call `intersection` without any arguments; this returns a
+copy of the set `S`.
+
+```python
+set([1, 2]).intersection([2, 3])             # set([2])
+set([1, 2, 3]).intersection([0, 1], [1, 2])  # set([1])
+```
+
+<a id='set·intersection_update'></a>
+
+### set·intersection\_update
+
+`S.intersection_update(*others)` removes from the set `S` any elements found in
+any of the iterable sequences `*others`. It returns `None`.
+
+If `s` and `t` are sets, `s.intersection_update(t)` is equivalent to `s &= t`;
+however, note that the `&=` augmented assignment requires both sides to be sets,
+while the `intersection_update` method takes arbitrary iterable sequences.
+
+It is permissible to call `intersection_update` without any arguments; this
+leaves the set `S` unchanged.
+
+`intersection_update` fails if the set `S` is frozen or has active iterators.
+
+```python
+s = set([1, 2, 3, 4])
+s.intersection_update([0, 1, 2])       # None; s is set([1, 2])
+s.intersection_update([0, 1], [1, 2])  # None; s is set([1])
+```
+
+<a id='set·isdisjoint'></a>
+
+### set·isdisjoint
+
+`S.isdisjoint(x)` returns `True` if the set `S` and the iterable sequence `x` do
+not have any values in common, and `False` if they have at least one value in
+common.
```

Comment:
otherwise

(it's fewer words, and avoids the risk that the two predicates are not each other's exact negation)

ditto below

Reply:
Good point